frontend/src/views/ImageUploadView.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ImageUpload  from '../components/ImageUploader'
+
+export class ImageUploadView extends React.Component {
```

Comment:
We don't actually need view for image upload, because we will use it as component in other views. But you are right, that we need view to show this components, for which we have no real place now. So you can rename this View to example view and we can use it for showing new components.

Reply:
done! changed the name to ExampleView!